crates/rolldown_common/src/inner_bundler_options/types/module_type.rs

```diff
@@ -10,7 +10,7 @@ use serde::Deserialize;
   derive(Deserialize, JsonSchema),
   serde(rename_all = "camelCase", deny_unknown_fields)
 )]
-#[derive(Debug, Clone, Copy)]
```

Comment:
Can you explain why?

Reply:
I see the related pr, https://github.com/rolldown/rolldown/pull/1632, such a standalone pull request is confusing.

Reply:
Yeah. I'll add more descriptions in the future.